pkgs/data/misc/shared-mime-info/default.nix

```diff
@@ -1,5 +1,9 @@
-{ lib, stdenv
-, fetchurl
+{ stdenv
+, lib
+, fetchFromGitLab
+, fetchpatch
+, meson
+, ninja
 , pkg-config
 , gettext
 , perlPackages
```

Comment:
Do we actually even still need perl?

Reply:
Good point. perl seems to be present since the commit that introduced the file, 67d41efde183d84f967e350e3e75d8f8dedd165d . I'm not sure it was never needed though, as I can't find any perl scripts in the shared-mime-info git repo, neither in 2.1, nor in 0.21 which is the first version added in the commit referenced earlier. I'll remove it and try it out.

Reply:
@jtojnar seems everything still works even with perl removed.